en/Installation.md

```diff
@@ -112,30 +112,52 @@ In the following, the installation is documented for Ubuntu, Debian, Fedora, Cen
+includes only the Java dependencies JabRef uses. There are two major ways of
+obtaining JabRef for your platform.
 
-#### Ubuntu
+#### Building From Source
 
-1. Follow the steps at https://bell-sw.com/pages/liberica_install_guide-11.0.4/#apt-repository-deb-based-linux-distributions
-2. Ensure that gtk3 is available: `sudo apt-get install gtkterm`
+This method is mainly for package maintainers and users who would like to build
+the latest snapshots of JabRef directly from the source.
 
-Now, you should be able to execute `java -jar JabRef--master--latest.jar`
+To build JabRef from source, you first need to have a working Java Development
+Kit 11 (JDK 11) and Git installed on your system. After installing the two
+requirements, you open a terminal window (i.e., a command prompt) and type the
+following:
 
-#### Gentoo
+```
+git clone https://github.com/JabRef/jabref
+cd jabref
+git submodule update --init
+./gradlew assemble
+./gradlew jlink
+```
+
+In a nutshell, you clone the latest snapshot of JabRef into `jabref` directory,
+change directory to `jabref`, initialize and update all the submodules
+(dependencies) of JabRef, assemble them to be built via JDK 11, and finally
+build and link them together.
 
-Gentoo provides AdoptOpenJDK 11 in its package `dev-java/openjdk-bin`; it is not available for 32-bit systems!  This guide assumes that you have a system with working GUI, it was not tested on a headless system.
+The output should be the `build/image` subdirectory that contains the JabRef binary
+with all of its Java dependencies. To start JabRef, you need to run
+`bin/JabRefMain` (in Linux and MacOS) or `bin/JabRefMain.bat` (in Windows) under
+`build/image` subdirectory.
 
-Be aware that this guide changes your default system JVM, which is currently not supported by Gentoo devs and may cause problems with other Java-related packages!
+#### Using Prebuilt Binaries
 
-1. Unmask the package if you do not have unmasked the `~amd64` as a system default: `echo ">=dev-java/openjdk-bin-11.0.4_p11 ~amd64" >> /etc/portage/package.accept_keywords`
-2. Unmask the `gentoo-vm` useflag in order to select Java 11 as a system VM (currently not supported by Gentoo devs): `echo "dev-java/openjdk-bin -gentoo-vm" >> /etc/portage/profile/package.use.mask`
-3. Add the `gentoo-vm` useflag before installing the package: `echo ">=dev-java/openjdk-bin-11.0.4_p11 gentoo-vm" >> /etc/portage/package/use`
-4. Install: `emerge -av dev-java/openjdk-bin:11`
-5. View the list of available JDKs: `java-config -L`
-6. Select AdoptOpenJDK 11: `java-config -S openjdk-bin-11`
+This method is mainly for anyone who would like to download and run the latest
+snapshot of JabRef.
 
-Now, you should be able to execute `java -jar JabRef--master--latest.jar`.
+To use the prebuilt binaries, simply visit http://builds.jabref.org/master/ and
+download the packaged binaries (e.g., `dmg` files for MacOS, `deb` for
+Debian-based distributions, `rpm` for RedHat-based distributions and `exe` for
```

Comment:
We do not yet provide debs or rpms. So please remove this. Otherwise lgtm!

Reply:
Done